db/dao/sector_dao.py

```diff
@@ -25,3 +70,32 @@ def delete_sector(sector, commit=True):
         DB().session.delete(sector)
         if commit:
             DB().session.commit()
+
+    @staticmethod
+    def get_sectors_linked_to_file(file):
+        """Returns all the sectors where the file data
+        has been saved
+        :param file: file object model
+        """
+        return DB().session.query(Sector).filter_by(
+            file_id=file.id
+        ).all()
+
+    @staticmethod
+    def get_unused_sectors_count():
+        """
```

Comment:
We will need to modify this if `@property data` change is added.

Reply:
will we though... hehe